sample/app/build.gradle

```diff
@@ -0,0 +1,156 @@
+buildscript {
+    repositories {
+        jcenter()
+    }
+    dependencies {
+        classpath 'com.android.tools.build:gradle:2.2.0'
```

Comment:
It still uses 3.2.1, as we only have one gradle wrapper.
This should still work with older versions of course, but that's fixable with a new major release.